README.md

```diff
@@ -7,14 +7,27 @@ Gearbox.js
 [![Build Status](https://travis-ci.org/cognitedata/gearbox.js.svg?branch=master)](https://travis-ci.org/cognitedata/gearbox.js)
 [![codecov](https://codecov.io/gh/cognitedata/gearbox.js/branch/master/graph/badge.svg)](https://codecov.io/gh/cognitedata/gearbox.js)
 
-## Setup
+## Install package
 
-Run `yarn`.
+Run `yarn add @cognite/gearbox` or `npm install @cognite/gearbox --save`.
 
 Install additional dependencies:
```

Comment:
I would group the install scripts for the dependencies. I don't want to copy&paste 3 times. What about doing `yarn add @cognite/sdk antd styled-components` and the same for npm?

Reply:
Please check updated section at the readme branch